### ISSUE TITLE
Min & Max

### ISSUE THREAD

**Issue by kempeng** [shopspring/decimal#113]
I understand the reasons why the Min Max functions for decimal are implemented as they are: to allow for evaluation of multiple values.

However, would it not be useful to also have versions that are similar in syntax as d.Add(d2) or other math functions, and implement also:
d.Min(d2)
d.Max(d2) ?

**Comment by mwoss** [comment]
@kempeng I'm not a fan od that idea. API would look messy imo. We are considering adding helper functions like `GreaterThan`, `LesserThan`

**Comment by lovung** [comment]
I think they have already existed at https://github.com/shopspring/decimal/blob/66332b32a9d5bfdf6cfcf6c997a5b5cae39dccb9/decimal.go#L1173-L1205

**Comment by mwoss** [comment]
Yes, you are right but they are not bound to the Decimal struct itself. @kempeng asked about `Min` and `Max` methods for decimal struct.

**Comment by lovung** [comment]
@mwoss I don't think it's necessary and I think you can close this issue 👍
